docs/source/reference/compilation.rst

```diff
@@ -102,6 +102,20 @@ JIT functions
 
       .. seealso:: :ref:`architecture`
 
+   .. method:: inspect_llvm(signature=None)
+
+      Return a dictionary keying compiled function signatures to the human 
+      readable llvm byte code generated for the function.  If the signature 
```

Comment:
Nit: I would capitalize "LLVM". Also, it's really an IR (or intermediate representation); "human readable byte code"s sounds a bit contradictory.